go-controller/pkg/ovn/namespace.go

```diff
@@ -236,9 +260,11 @@ func (oc *Controller) createNamespaceLocked(ns string) *namespaceInfo {
 	defer oc.namespacesMutex.Unlock()
 
 	nsInfo := &namespaceInfo{
-		addressSet:       make(map[string]string),
-		networkPolicies:  make(map[string]*namespacePolicy),
-		multicastEnabled: false,
+		addressSet:              make(map[string]string),
+		networkPolicies:         make(map[string]*namespacePolicy),
+		hybridOverlayExternalGW: nil,
```

Comment:
Don't need to set to `nil` here. Just leave them blank they are nil by default.

Reply:
ok, i tend to prefer to explicitly initialize vars